scripts/migrate_operations_to_workflows.py

```
@@ -0,0 +1,277 @@
+        if self.dry_run:
+            logger.info(f"DRY RUN: Would migrate {len(items)} items")
+            return len(items), 0
+        
+        try:
+            with self.target_table.batch_writer() as batch:
+                for item in items:
+                    try:
+                        # Copy item as-is (structure is identical)
+                        batch.put_item(Item=item)
+                        success_count += 1
+                    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)